### DISCUSSION TITLE
Xinclude limitations

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#555] [Q&A]
Hi Patrick,

I am perhaps misusing the `<xi:include />` tag, I noticed that I can load font, graphic and color definitions but I can't load page definitions. Is that the intrinsic limit?

Thanks... 🙏

**Comment by pgundlach** [comment]
This should not be the case. XInclude in handled before the main processing sees the XML file, so there should not be a limit. (I do know however that there could be a bug - as always). 

Make sure that page definitions are seen by the publisher before the pages are created.

**Comment by Cicorione** [comment]
I moved the `Xinclude` tag right after the `layout` tag and yet isn't loaded... 🤷‍♂️

**Reply by pgundlach** [reply]
This is still unresolved? You could send me the complete project, if you want, and I'll take a look.

**Reply by Cicorione** [reply]
Hi Patrick, I haven't had time to prepare a very basic layout and verify if the issue still persists. I am going to try it ASAP, thanks!
